x/ccv/consumer/keeper/relay.go

```diff
@@ -239,7 +239,7 @@ func (k Keeper) OnAcknowledgementPacket(ctx sdk.Context, packet channeltypes.Pac
 	return nil
 }
 
-func (k Keeper) OnTimeoutPacket(ctx sdk.Context, packet channeltypes.Packet, data ccv.SlashPacketData) error {
+func (k Keeper) OnTimeoutPacket(ctx sdk.Context, packet channeltypes.Packet) error {
```

Comment:
Why is this function even in here?

Reply:
That's the same function that was removed from https://github.com/cosmos/interchain-security/pull/550/files/f6cec81f2b7022021698cf09c45d928dfd8f85a0#r1040203683.

It's a no-op. it can be removed.

Reply:
IMO, we should remove it.